.gitlab-scripts/publish_image.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
 
-TAG=$1
-cp ${@:2} Dockerfiles/agent
-docker build -t stackstate/stackstate-agent:$TAG -t stackstate/stackstate-agent:latest Dockerfiles/agent
+IMAGE_TAG=$1
+IMAGE_REPO=$2
+ARTIFACT_PATH=$3
+
+echo $IMAGE_TAG
+echo $IMAGE_REPO
+echo $ARTIFACT_PATH
+
+cp $ARTIFACT_PATH/*.deb Dockerfiles/agent
+
+docker build -t stackstate/${IMAGE_REPO}:${IMAGE_TAG} Dockerfiles/agent
+docker tag stackstate/${IMAGE_REPO}:${IMAGE_TAG} stackstate/${IMAGE_REPO}:latest
 docker login -u $DOCKER_USER -p $DOCKER_PASS
-docker push stackstate/stackstate-agent:$TAG
-docker push stackstate/stackstate-agent:latest
+echo docker push stackstate/${IMAGE_REPO}:${IMAGE_TAG}
+docker push stackstate/${IMAGE_REPO}:${IMAGE_TAG}
+echo docker push stackstate/${IMAGE_REPO}:latest
```

Comment:
its a bit misleading that for dev builds, the latest is the latest that was build (from any branch). I would expect this to be the latest master build

Reply:
agree